source/docs/api-reference/device-specific/talonfx/basic-pid-control.rst

```diff
@@ -12,7 +14,8 @@ Position closed loop is currently supported for all base :ref:`control output ty
 
 In a Position closed loop, the gains should be configured as follows:
 
-- :math:`K_s` - unused, as there is no target velocity
+- :math:`K_g` - output to overcome gravity (output)
+- :math:`K_s` - Velocity Sign: unused; Closed-Loop Sign: output to overcome static friction (output)
```

Comment:
kS with velocity sign is used in position closed loop mode if the velocity is specified

Reply:
That's a separate section further down the page (Motion Profiling)